src/util/InstanceCount.h

```diff
@@ -4,12 +4,28 @@
 #define LSST_QSERV_UTIL_INSTANCECOUNT_H
 
 // System headers
+#include <atomic>
 #include <map>
 #include <mutex>
 #include <string>
 
 namespace lsst::qserv::util {
 
+class InstanceCount;
+
+class InstanceCountData {
```

Comment:
If you don't have a use of this class in the public interface, then you may do:
```
class InstanceCountData;
```
Then push the class into the CC file to make the API of `InstanceCount` clean of the implementation-specific artifacts.